.github/workflows/ci.yaml

```diff
@@ -0,0 +1,71 @@
+on: [push, pull_request]
+
+name: CI
+env:
+  CARGO_TERM_COLOR: always
+  RUSTFLAGS: -D warnings
+  RUSTDOCFLAGS: -D warnings
+
+jobs:
+  Build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Install libasound2-dev
+      run: |
+        sudo apt-get update
+        sudo apt-get install libasound2-dev
```

Comment:
Huh ... what's this for?

Reply:
The example code uses the [`cpal`](https://crates.io/crates/cpal) crate, which needs `libasound2-dev` to be installed in order to compile.

Reply:
It's only a dev-dependency, so users of our crate won't need it.

Reply:
Oh for sure. I was just hoping we could remove it from CI :-) But we can't since we `--all-targets` (which is good - we want those examples to keep compiling)